### PR TITLE
Fixed issues below

### PR DESCRIPTION
**Pull request by Manarnz**
Reimplement clear results and configs in analysis mode
The slider does not change sat val when EVO is off and result is selected

### CODE REVIEW

**Comment by YesugenB** [comment]
The fonts for function type and initial satisfaction value become smaller when you press clear in analysis view?

**Review by YesugenB — DISMISSED** [review]
Looks great!

**Review by YesugenB — DISMISSED** [review]

**Review by Domily11 — CHANGES_REQUESTED** [review]
I thought the font looked a bit odd when it is in EVO mode, did you put an outline around the text?

File: leaf-ui/js/object/evoObjects.js [hunk 1]
```diff
@@ -145,33 +145,26 @@ class EVO {
...
-        } else {
-            if (analysisResult !== undefined) {
-                if (analysisResult.get('selected')) {
-                    EVO.displaySlider(true);
-                }
-                else {
-                    EVO.displaySlider(false);
-                }
+        }
+        else {// If EVO is off
```
Comment (angelac2023) [inline leaf-ui/js/object/evoObjects.js]: Can you put this comment above?

File: leaf-ui/js/object/evoObjects.js [hunk 2]
```diff
@@ -145,33 +145,26 @@ class EVO {
...
+            if (analysisResult !== undefined && analysisResult.get('selected')) {// If a result is selected
```
Comment (angelac2023) [inline leaf-ui/js/object/evoObjects.js:163]: See previous line.

**Review by angelac2023 — COMMENTED** [review]
Other than the comment uniformity, looks good!

**Review by angelac2023 — APPROVED** [review]

**Review by amgrubb — APPROVED** [review]
Nice work!

**Pull request merged**